README.md

```diff
@@ -21,57 +23,44 @@ For more information about Metrilo and its features please visit [the official w
 
 :information_source: If you decide to follow along, you'll end up with some resources that you (most probably) won't use in the future. However, that's perfectly fine - after you're done testing, you can just ping us and we'll reset your project to a pristine state.
 
-Once you've [signed up for a Metrilo account](https://app.metrilo.com/signup), navigate to the `Settings -> Installation` page in your [Metrilo project](https://app.metrilo.com) and keep your `API Token` handy - you'll need it for the rest of this tutorial:
+Once you've [signed up for a Metrilo account](https://app.metrilo.com/signup), navigate to the `Settings -> Installation` page in your [Metrilo project](https://app.metrilo.com) and keep your `API Token` and `API Secret` handy - you'll need them for the rest of this tutorial:
 
 ![](images/api-token.png)
 
 Now you need to import your shop's resources into Metrilo - categories, products, customers, orders. Then you can start registering actions on them - views, adds to cart, new orders.
 
-Let's create a category, a product and a customer. We'll assume
-the current time (in milliseconds since 1970) is `1518004715732`:
+Let's create a category, a product and a customer.
+We'll assume that: 
+- the current time (in milliseconds since 1970) is `1518004715732`
+- the API token of your Metrilo project is `sh0p-t0k3n`
+- the API secret of your Metrilo project is `m3trilo-secr3t`
 
-```
-curl -X POST "https://trk.mtrl.me/category" -i -H "Content-Type: text/plain" -d \
-'{
-  "time": 1518004715732,
-  "token": "sh0p-t0k3n",
-  "params": {
-    "id": "1",
-    "name": "Awesome clothing",
-    "url": "https://my-awesome-shop.com/awesome-clothing"
-  }
-}'
-```
+:warning: v2 of the Metrilo API uses a custom `X-Digest` header, which is a HMAC-SHA256 digest generated by using the request body string as the message and your Metrilo API secret as the key. You have to generate separate digest for every resource call you make. 
```

Comment:
Lets drop `v2` from here. `Metrilo API` will be the latest version, all others previous ones will have a version number metioned. Make it easy for customers to integrate with the latest, and say it like there are no other APIs. We want customers to integrate and use the latest.
```suggestion
: information_source: Metrilo API uses a custom `X-Digest` header, which is a HMAC-SHA256 digest generated by using the request body string as the message and your Metrilo API secret as the key. You have to generate separate digest for every resource call you make. 
```